config/config.go

```diff
@@ -84,13 +85,13 @@ func ShouldInclude(name string, includeREs []Expression, excludeREs []Expression
 	// If no rules are defined, should always include
 	if len(includeREs) == 0 && len(excludeREs) == 0 {
 		return true
-	// If a rule that exclude matches, should not include
+		// If a rule that exclude matches, should not include
 	} else if matches(name, excludeREs) {
 		return false
-	// Given the 'name' is not in the 'exclude' list, should include if there is no 'include' list
+		// Given the 'name' is not in the 'exclude' list, should include if there is no 'include' list
```

Comment:
NIT: same here - seems to have added some extra spacing before the comments

Reply:
Ah yea this is not good. Again, I think this was caused by my upgrade to go 1.17. It looks like this is now the new standard formatting. I moved the comments around to make it more readable: f1cf9600d599cd669a482881bc62178416d43928

Reply:
perfect! thanks for addressing it